### ISSUE TITLE
apidoc-swagger command isn't working

### ISSUE THREAD

**Issue by jbrauchler** [fsbahman/apidoc-swagger#7]
When I try to run the apidoc-swagger command with my api I am getting the error 
`env: node\r: Not a directory`
I am following the apidoc commands and have had apidoc working properly so I am not sure what the issue is. Any help would be great.

**Comment by fsbahman** [comment]
Hi, You closed the issue, is it still there or you find a way around it?
I was not around didn't have a time to look at it.

**Comment by jbrauchler** [comment]
I believe I got it working, but I had run it as 
apidoc swagger rather than apidoc-swagger. I just wasn't able to find the swagger Jason schema from there.

**Comment by jbrauchler** [comment]
I thought I had it working but it turns out I do not.

**Comment by ramkumarc** [comment]
I have the same issue as well. Command and response as given below

```
apidoc-swagger -i api/src/ -o public/docs/
: No such file or directory
```

**Comment by shahzaibalikhan** [comment]
Facing same issue here... any update on this?

**Comment by jbrauchler** [comment]
I still haven't found a fix I'm not sure where the error is and haven't heard anything from fsbahman.

**Comment by jbrauchler** [comment]
Honestly, I have just been using apidoc, I find that the format looks nicer using apidoc.

**Comment by fsbahman** [comment]
Hi, May I know how do you install it and how your directory structure look like?

This is a very simple way to see if it works:
1. install apidoc-swagger:  `npm install apidoc-swagger -g`
2. cd into a directory with a file(s) you want to create documentation out of it. One remark, by default only  known file extensions will be analyzed by apidoc like: `*.js, *.c, *.java, *.cs` 
3. just run `apidoc-swagger` without any parameters, it should create a sub directory `doc` containing a file `swagger.json`

Let me know if this is working for you.

**Comment by jbrauchler** [comment]
I attempted that, it does not work either.

**Comment by fsbahman** [comment]
What kind of error did you get? 
Could you give me more information about the environment you are running your application?

**Comment by jbrauchler** [comment]
I got the same error as with parameters. I'm running on Mac, I am experienced with apidoc. That's really about it nothing special about my environment.

**Comment by lfyzjck** [comment]
I has the same problem on Mac. Perhaps the problem is file formating:

https://github.com/mllrsohn/nw-builder/issues/75

Just edit `bin/apidocSwagger.js` in vim and run `set ff=unix`.

**Comment by nzoss** [comment]
That indeed seems to be the issue. Resaving the file locally with unix line breaks worked file.

**Comment by hareesh-blippar** [comment]
The issue is with using windows newline characters (\r\n instead of \n) in the source file.

@fsbahman Could you update bin/apidocSwagger.js to use /n returns instead of /r/n returns ?

**Comment by fsbahman** [comment]
I'm using windows and tried to change line endings, after commit there is no history for them but it looks like changed. I've tried to download the package on my Ubuntu VM and it work fine, for now I don't have access to a mac to test it there as well. If someone had a look let me know if it is fine.

**Comment by cadesalaberry** [comment]
@fsbahman I think this issue can now be closed.

**Comment by fsbahman** [comment]
OK cool, thanks.

**Comment by felipe-augusto** [comment]
@fsbahman This issue is back in the new version (0.3.0), can you please check? I reverted to 0.2.2 and everything works fine.

**Comment by fsbahman** [comment]
@felipe-augusto Well actually version 0.3.0 is not directly related to previous versions. I created that one for the effort made by @kukicvladimir in this PR: https://github.com/fsbahman/apidoc-swagger/pull/5 
since changes are a lot with so many conflicts and I didn't have time to merge it, I've decided to release it from the code that is not in the master branch, I expected some backward compatibility issues. I will try to bring nice fix and features from that branch to the main branch soon.
